### PR TITLE
Update readme for getAllIanaWindowsMap method

### PR DESCRIPTION
**Pull request by funkydev**
It updates README file for `getAllIanaWindowsMap()` method.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,3 +72,12 @@ import { findAlias } from "windows-iana";
 const result = findAlias("America/New_York");
 console.log(result); // ["America/New_York", "US/Eastern"]
 ```
+
+## `getAllIanaWindowsMap()`
+
+```
+import { getAllIanaWindowsMap } from "windows-iana";
+
+const result = getAllIanaWindowsMap("America/New_York");
```
Comment (rubenillodo) [inline README.md]: This line doesn't seem correct.

```suggestion
const result = getAllIanaWindowsMap();
```

Comment (funkydev) [reply README.md]: I fixed that :)

**Comment by funkydev** [comment]
@rubenillodo Could you make the release after merge the current PR?

**Comment by funkydev** [comment]
@rubenillodo Thank you! 🎉

**Comment by funkydev** [comment]
@rubenillodo Is something wrong with that PR? Travis CI has status `Expected — Waiting for status to be reported` since yesterday. 😞

**Pull request merged**